### DISCUSSION TITLE
export excel failed

### DISCUSSION THREAD

**Discussion by theone-daxia** [orchidsoftware/platform#2627] [Q&A]
Hi
i want to export an excel file, but there is a circle on the button that keeps spinning, and the file was not downloaded.
here is my code and screenshot.

```
public function actionExport(Request $request)
{
    $this->validate($request, [
        'start_time' => 'required',
        'end_time' => 'required',
    ]);
    $startTime = $request->start_time;
    $endTime = $request->end_time;

    return Excel::download(new exportOrder($startTime, $endTime), 'orders.xlsx');
}
```

<img width="623" alt="download" src="https://github.com/orchidsoftware/platform/assets/20626884/597ff6fc-1a00-4c54-858e-191e9909284c">

wish to reply.
thanks!

**Comment by c-gross** [comment]
Check the response in the browser dev tools. There might be an error.
Check log files in orchid.
